src/3.02/3.02.yaml

```diff
@@ -0,0 +1,25 @@
+---
+id: '3.02'
+name: vpc_flows_logging_disabled
+# Refer to corresponding queries: 3_02_vpc_flows_logging_disabled.{sql|yaral|...}
+display_name: Disabling VPC Flows logging
+category: Cloud Provisioning Activity
+severity: Medium
+attack_mapping:
+  - technique: T1562.008
+    title: Impair Defenses (Disable Cloud Logs)
+    link: https://attack.mitre.org/techniques/T1562/008/
+description: |
+  Disabling VPC Flows logging.
+
+sources:
+  - Audit Logs - Admin Activity
+
+use_cases:
+  - Detect
+  - Audit
+
+test:
```

Comment:
Not required for this PR. But consider adding event generation steps in this or subsequent PR. 
Here's an example:
- YAML: https://github.com/GoogleCloudPlatform/security-analytics/blob/main/src/1.01/1.01.yaml#L22
- Auto-generated README: https://github.com/GoogleCloudPlatform/security-analytics/blob/main/src/1.01/1.01.md

In this case, the following gcloud command disable vpc flow logs for a subnet:
`gcloud compute networks subnets update #{subnet-name} --no-enable-flow-log`

Reply:
Added in commit: https://github.com/GoogleCloudPlatform/security-analytics/pull/11/commits/ac4389335e5d86d65e9584b7b20bf9589d2c1269